src/fix_langevin.cpp

```diff
@@ -274,21 +306,122 @@ void FixLangevin::init()
   if (strstr(update->integrate_style,"respa"))
     nlevels_respa = ((Respa *) update->integrate)->nlevels;
 
-  if (gjfflag) gjffac = 1.0/(1.0+update->dt/2.0/t_period);
+  if (strstr(update->integrate_style,"respa") && gjfflag)
```

Comment:
using `strstr()` can lead to false positives. it is safer to use `utils::strmatch(modify->fix[i]->style,"^respa")` here (and in similar cases).